### PR TITLE
Log Site start to -log. Fix for #1205

### PR DESCRIPTION
**Pull request by tobya**
This is a 1 line fix for #1205. 

Details of each site started will be written to Log as well as stdout.

I dont think this needs a test case.

### CODE REVIEW

File: caddyhttp/httpserver/server.go [hunk 1]
```diff
@@ -363,6 +363,8 @@ func (s *Server) OnStartupComplete() {
 			output += " (only accessible on this machine)"
 		}
 		fmt.Println(output)
+		//Also print to -log.
```
Comment (mholt) [inline caddyhttp/httpserver/server.go]: This comment isn't interesting. :) Let's remove it.

File: caddyhttp/httpserver/server.go [hunk 2]
```diff
@@ -363,6 +363,8 @@ func (s *Server) OnStartupComplete() {
...
+		log.Println(output)
```
Comment (mholt) [inline caddyhttp/httpserver/server.go]: Do you want to suppress this output if Caddy is running quietly?

Comment (tobya) [inline caddyhttp/httpserver/server.go]: I'm not sure I understand.  By my basic testing if I run `caddy -quiet` I get no output. 

If I run `caddy -log stdout -quiet` I get the Startup and SIGINT log but not the output of line 367

Am I missing something?

**Comment by mholt** [comment]
Thanks Tobya!

**Pull request merged**